couler/core/syntax/__init__.py

```diff
@@ -22,3 +22,6 @@
     add_volume,
     create_workflow_volume,
 )
+from couler.core.syntax.image_pull_secret import (  # noqa: F401
+    add_image_pull_secret,
+)
```

Comment:
please update this

Reply:
sorry, this is my first time submitting pr in github, how should i fix it?

Reply:
look like done. please check, thanks